corehq/ex-submodules/soil/templates/soil/partials/bootstrap3/download_in_progress.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
```

Comment:
that could be true but i believe this modal is used in a variety of places other than that, notably media download